packages/config/config/devices/0x0344/he-tps05.json

```
@@ -0,0 +1,970 @@
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				}
+			]
+		},
+		"72": {
+			"label": "Relay 2: Timer Mode Duration",
+			"description": "Used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				}
+			]
+		},
+		"73": {
+			"label": "Relay 3: Timer Mode Duration",
+			"description": "Used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				}
+			]
+		},
+		"74": {
+			"label": "Relay 4: Timer Mode Duration",
+			"description": "Used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				}
+			]
+		},
+		"75": {
+			"label": "Relay 5: Timer Mode Duration",
+			"description": "Used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				}
+			]
+		},
+		"101": {
+			"label": "Relay Inverse Mode, Inching - Group 1",
+			"description": "Combine the relay numbers to a two-digit number. Example: 24 = Relay 2 & 4",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 54,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		"102": {
+			"label": "Relay Inverse Mode, Inching - Group 2",
+			"description": "Combine the relay numbers to a two-digit number. Example: 24 = Relay 2 & 4",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 54,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		"141": {
+			"label": "Energy Consumption Report Interval",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 10
+		},
+		"142": {
+			"label": "Energy Consumption Report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"143": {
+			"label": "Sensors Consecutive Reporting Interval",
```

I think this is better
```suggestion
			"label": "All Sensors: Reporting Interval",
```